mep/accounts/admin.py

```diff
@@ -8,46 +8,10 @@
 
 from mep.accounts.models import Account, Address, Subscription,\
     Reimbursement, Event, SubscriptionType, Borrow, PartialDate, Purchase
-from mep.common.admin import NamedNotableAdmin, CollapsibleTabularInline
+from mep.common.admin import NamedNotableAdmin, CollapsibleTabularInline, AUTOCOMPLETE
 from mep.footnotes.admin import FootnoteInline
 
 
-# predefine autocomplete lookups (most are used on more than one form)
```

Comment:
This didn't really need to be moved, since the bibliography autocomplete url is only needed in the footnote admin and not used anywhere else - but I'm fine with it.